upgrading/rosa-hcp-upgrading.adoc

```diff
@@ -6,20 +6,24 @@ include::_attributes/attributes-openshift-dedicated.adoc[]
 
 toc::[]
 
-[id="rosa-hcp-lifecycle-policy_{context}"]
-== Life cycle policies and planning
+Upgrade {hcp-title-first} clusters by individually upgrading the hosted control plane and the machine pools with the ROSA command line interface (CLI), `rosa`.
```

Comment:
Suggestion to frame this abstract as a capability rather than an imperative:
```suggestion
You can upgrade {hcp-title-first} clusters by individually upgrading the hosted control plane and the machine pools with the ROSA command line interface (CLI), `rosa`.
```